pkg/sdn/plugin/controller.go

```diff
@@ -457,8 +461,17 @@ func (plugin *OsdnNode) AddServiceRules(service *kapi.Service, netID uint32) {
 	glog.V(5).Infof("AddServiceRules for %v", service)
 
 	otx := plugin.ovs.NewTransaction()
+	action := getServiceRuleAction(netID)
+
+	// Add blanket rule allowing subsequent IP fragments
+	if netID != osapi.GlobalVNID {
+		baseFragRule := generateBaseServiceFragRule(service.Spec.ClusterIP)
```

Comment:
@danwinship It's a problem for TCP too, no?  I haven't checked though, but any fragmented packet will hit this issue.